### DISCUSSION TITLE
Track Selection not working

### DISCUSSION THREAD

**Discussion by ahmadlimbada** [GrapesJS/grapesjs#4516] [Q&A]
Hi @artf and @lexoyo,

I tried setting trackSelection to False in init but still it's tracking the selection.

![image](https://user-images.githubusercontent.com/30268583/185143222-4d34a542-7ce6-412f-80de-ace041be59a8.png)

but still it's tracking the selection and showing borders around previously selected components
![image](https://user-images.githubusercontent.com/30268583/185143915-9f89228b-4dac-402b-9380-c2b8803a2756.png)

**Comment by artf** [comment]
I see no issues with that option, so if you believe there is an issue, please create a reproducible demo first.
